File: CPP/film.cpp
#include <string>

using namespace std;

//class cinema film
class Film {
private:
    int id;
    string judul;
    string genre;

public:
    Film() {
        id = 0;
        judul = "";
        genre = "";
    }

    Film(int id, string judul, string genre) {
        this->id = id;
        this->judul = judul;
        this->genre = genre;
    }

    int getId() const { return id; }
    string getJudul() const { return judul; }
    string getGenre() const { return genre; }

    void setId(int id) { this->id = id; }
    void setJudul(string judul) { this->judul = judul; }
    void setGenre(string genre) { this->genre = genre; }

    virtual ~Film() {
    }
};

File: Python/film.py
# class cinema film
class Film:
    def __init__(self, id=0, judul="", genre=""):
        self.__id = id
        self.__judul = judul
        self.__genre = genre

    # Getter
    def getId(self):
        return self.__id

    def getJudul(self):
        return self.__judul

    def getGenre(self):
        return self.__genre

    # Setter
    def setId(self, id):
        self.__id = id

    def setJudul(self, judul):
        self.__judul = judul

    def setGenre(self, genre):
        self.__genre = genre